### ISSUE TITLE
Upload Wireframe to repository

### ISSUE THREAD

**Issue by Henrique-mazzoleni** [The-Ironsquad/Tender#2]
link the wireframe from the mvp which was discussed on 12.09.2023 to the repo.

**Comment by Henrique-mazzoleni** [comment]
I saw that you added the wireframe to the readme file @zefanjaaa but before we close this issue I would like to ask if it wouldn't it be better for the wireframe to be in it's own file on a designs folder perhaps. What do you think?

**Comment by hiranimotors** [comment]
I have changed the name of the default branch to `develop`. I have checked and it is not possible to push local changes directly to `develop`. Attempting to do so throws a rule violation error. I think we are more or less set up to start developing now. I want to get some approvals on some PRs to check merge functionality. Will get some approvals this evening so that I can verify it's all working as it should.
